### PR TITLE
Classifier: Convert drawing stories to CSF

### PR DESCRIPTION
**Pull request by eatyourgreens**
Remove `storiesOf` from the drawing tool stories and replace them with [Component Story Format](https://storybook.js.org/docs/react/api/csf) functions.

Package:
lib-classifier

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?

## Components
- [ ] Has a storybook story been created or updated?
- [ ] Is the component accessible? 
  - [ ] Can it be used with a screen reader? [BBC guide to testing with VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
  - [ ] Can it be used from the keyboard? [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - [ ] Is it passing accessibility checks in the storybook?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `yarn panic && yarn bootstrap` or `docker-compose up --build` and app works as expected?

## Publishing

- [ ] Is the changelog updated?
- [ ] Are the dependencies updated for apps and libraries that are using the newly published library?

## Post-merging

- [ ] Did the app deploy to https://frontend.preview.zooniverse.org/projects/:project-name/:owner or https://frontend.preview.zooniverse.org/about?
- [ ] Is the new feature working or bug now fixed?
  - [ ] Is there a Talk or blog post written to announce the new feature(s)?
- [ ] Is the design working across browsers (Firefox, Chrome, Edge, Safari) and mobile?
  - [ ] Is this approved by our designer?
- [ ] Is this ready for production deployment?

### CODE REVIEW

**Comment by eatyourgreens** [comment]
This seems to have fixed the Transcription Line stories.

Screenshot of the transcription line stories with an open text popup.

File: packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.js [hunk 1]
```diff
@@ -98,7 +98,7 @@ function InteractionLayer ({
       activeTool.deleteMark(activeMark)
       setActiveMark(undefined)
     } else {
-      activeMark.setSubTaskVisibility(true, node)
+      activeMark?.setSubTaskVisibility(true, node)
```
Comment (eatyourgreens) [inline packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.js:101]: I'm not sure why, but `activeMark` is undefined here in the stories.

Comment (eatyourgreens) [reply packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.js:101]: Figured it out: the first Transcription Line story calls `transcriptionLine.finish()` when the line isn't active. In a real drawing task, a mark is always active when receiving events, so only active marks can be finished.

**Comment by eatyourgreens** [comment]
@srallen I figured out why the subtasks weren't showing for drawing stories, so I've fixed that here. The code is kind of ugly, but works. I think the API for setting subtask visibility changed at some point, but the stories weren't updated.

**Comment by eatyourgreens** [comment]
I've adjusted the `TranscriptionLine` stories so that `activeMark`, `finished` and `subtask` should show up as toggles in the Controls panel.

File: packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.js (hunk 1 above)
Comment (eatyourgreens) [reply packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.js:101]: Fixed by adding controls to each story, but the existence check may still be useful.

**Comment by eatyourgreens** [comment]
Converting the stories to functions exposed a bug where the store crashes if you run a story twice, because you can't pass a new store into an existing Provider. Fixed by running `setUpStores()` once for each story, then updating the stores via actions in a `updateStores` function.

**Comment by srallen** [comment]
@ErikOstlund do you have time to review this one?

**Review by ErikOstlund — APPROVED** [review]
All unit test are passing. All drawing tools work. Looks good to me.

**Pull request merged**